datadog_checks_base/requirements.in

```diff
@@ -16,5 +16,6 @@ requests-kerberos==0.12.0
 requests_ntlm==1.1.0
 simplejson==3.6.5
 six==1.12.0
+typing==3.7.4.1
```

Comment:
We could/should have used a `; python_version < '3.0'` specifier (although in practice installing the `typing` backport on Python 3+ should be a no-op), so…

```pip
typing==3.7.4.1; python_version < '3.0'
```